hw/virtio/vhost-user-sound.c

```diff
@@ -149,6 +149,40 @@ static void vus_snd_handle_output(VirtIODevice *vdev, VirtQueue *vq)
      */
 }
 
+static void vhost_sound_guest_notifier_mask(VirtIODevice *vdev, int idx,
+                                            bool mask)
+{
+    VHostUserSound *vvc = VHOST_USER_SOUND(vdev);
```

Comment:
for consistency, maybe to use same variable naming convention like the others?

Reply:
Yes, you are right. I will rename it for consistency.